### DISCUSSION TITLE
Mentionner qu'il faut activer la fonctionnalité "Ingérations et aperçus de lien"

### DISCUSSION THREAD

**Discussion by FlorentPoinsaut** [Crownicles/Crownicles#1762] [Idées acceptées / déjà implémentées]
Bonjour,

Merci beaucoup pour ce super bot !
Je viens de le mettre en place et j'ai, dans un premier temps, été un peu supris du fait que beaucoup de commande affichait une réponse vide.

Comme ceci :
![image](https://user-images.githubusercontent.com/1256948/215446413-3d7abdd3-9bab-44f0-b423-30de482f213d.png)

Je viens de trouver qu'il fallait activer la fonctionnalité "Ingérations et aperçus de lien" :
![image](https://user-images.githubusercontent.com/1256948/215446080-ccabc9a8-f1ab-4575-9061-7d17e4160d35.png)

Ce qui donne un résultat cette fois correct : 
![image](https://user-images.githubusercontent.com/1256948/215446615-280c8cbc-8de9-49b1-84bd-c625bcd61008.png)

A moins que je sois passé à côté dans le doc, je n'ai pas vu mention du fait qu'il faille activer cette option.
Ai-je manqué de vigilance ? Est-ce que ça doit être rajouté dans la doc ?

**Comment by BastLast** [comment]
Bonjour merci pr le signalement je converti cela en issue je pense que c'est encore une  nouveauté de discord sur laquelle ils ont peu communiqué et qui est très pénible à gérer.

**Comment by FlorentPoinsaut** [comment]
Bonjour @BastLast 
Wahou quelle réactivité !
Ca marche, merci !
